ci/tools.sh

```diff
@@ -24,13 +24,16 @@ retry() {
     return $result
 }
 
-
 if retry rustup component add rustfmt ; then
     cargo fmt --all -- --check
 fi
 
 if retry rustup component add clippy ; then
-    cargo clippy --all --tests --features serde,rayon -- -D clippy::all
+    cargo clippy --all --tests --features serde,rayon -- -D warnings
+fi
+
+if command -v taplo ; then
```

Comment:
Decided to make this an "optional" check in this script so that people don't need to necessarily have `taplo` installed to have the script pass, although it will still be enforced on CI, where it will be installed.